### PR TITLE
fix #246 + BC layer for SF < 2.7

### LINKED ISSUE #246: TranslatedEntityType => could not be converted to string
Hi,

It seems that the "translation_property" doesn't work. I'm using a2lix/TranslationFormBundle
 3.x version with Symfony v3.0.9

I tried the v2.x and it return an error "could not load type entity'.

```
->add('tags', TranslatedEntityType::class, array(
                    'class' => 'BlogBundle\Entity\Tag',
                    'translation_property' => 'name',
                    'query_builder' => function(\Doctrine\ORM\EntityRepository $er) {
                        return $er->createQueryBuilder('t')
                                  ->orderBy('t.name', 'ASC')
                                  ->andWhere('t.active = 1');
                    },
                    'expanded'        => true,
                    'multiple'           => true,
                    'label'                => false,
                    'error_bubbling' => true,
                ))
```

error:

```
Catchable Fatal Error: Object of class BlogBundle\Entity\Tag could not be converted to string 
```

Stack Trace:

```
in vendor\symfony\symfony\src\Symfony\Bridge\Doctrine\Form\Type\DoctrineType.php at line 68
```

It works with a __toString() method, but, it's a little "old school" .. no ? :)

Thanks for your help.

### PR DESCRIPTION
**Pull request by LanmanGroup**
I am targeting this branch, because this is BC.

Closes #246 

## Changelog

```markdown
### Fixed
- `TranslatedEntityType` is now compatible with `choice_label` for Symfony >= 2.7
```

## Subject

`choice_label` is there since Symfony 2.7

### CODE REVIEW

**Comment by jordisala1991** [comment]
Please fix travis. You will need to rebase on 2.x

File: Form/Type/TranslatedEntityType.php [hunk 1]
```diff
@@ -45,7 +51,7 @@ public function configureOptions(OptionsResolver $resolver)
                     ->select('e, t')
                     ->join('e.translations', 't');
             },
-            'property' => function (Options $options) {
+            $optionProperty => function (Options $options) {
```
[the remark this answers is not in the document]
Comment (LanmanGroup) [reply Form/Type/TranslatedEntityType.php]: I don't' change the original code, I fixed only the the name of the option

[the remark this answers is not in the document]
Comment (jordisala1991) [reply Form/Type/TranslatedEntityType.php]: That's why I am asking. If you didn't check if it works or not, we can't be sure.

[the remark this answers is not in the document]
Comment (LanmanGroup) [reply Form/Type/TranslatedEntityType.php]: In my current project with SF 3.x works good.

**Pull request merged**